espree.js

```diff
@@ -368,8 +534,14 @@ function parse(code, options) {
         if (extra.loc) {
             acornOptions.locations = true;
         }
-    }
 
+        if (extra.ecmaFeatures.jsx) {
+            if (extra.ecmaFeatures.spread !== false) {
```

Comment:
I don't think this is necessary. In original Espree, we just check to see if jsx is true and we are inside a jsx element when an ellipsis token is found.